src/renderer/index.css

```
@@ -82,32 +90,226 @@
     @apply border-border;
   }
 
+  html {
+    /* Crisp font rendering on Linux/Wayland (sub-pixel AA off by default) */
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    text-rendering: optimizeLegibility;
```

Stylelint (17.12.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)